docs/source/reference/interactive-nodes.rst

```diff
@@ -86,6 +86,16 @@ Interactive nodes can be stopped, restarted, and terminated, like any other clus
     $ # Terminate entirely:
     $ sky down sky-gpunode-<username>
 
+.. note::
+
+    Stopping a cluster does not lose data on the attached disks (billing for the
+    instances will stop while the disks will still be charged).  Those disks
+    will be reattached when restarting the cluster.
+
+    Terminating a cluster will delete all associated resources (all billing
+    stops), and any data on the attached disks will be lost.  Terminated
+    clusters cannot be restarted.
+
```

Comment:
For the `Getting multiple nodes` below, I would prefer to switch the suggested way to:
`sky launch -c my-cluster --num-nodes 16 --gpus V100:8` to make it easier to create interactive cluster.

Reply:
Done.